rdr_service/dao/workbench_dao.py

```diff
@@ -850,6 +850,48 @@ def _validate(self, resource_json):
                         raise BadRequest(
                             f"Invalid nonAcademicAffiliation: {institution.get('nonAcademicAffiliation')}")
 
+    def get_redcap_audit_researchers(self, **filters):
+        last_snapshot_id = filters.get('last_snapshot_id', None)
+        snapshot_id = filters.get('snapshot_id', None)
+        user_source_id = filters.get('user_source_id', None)
```

Comment:
```suggestion
    def get_redcap_audit_researchers(self, last_snapshot_id=None, snapshot_id=None, user_source_id=None):
```
nit: Using the parameters defaulted as None could avoid needing to explicitly unpack `**filters`

Reply:
smart!